src/editor.jsx

```diff
@@ -565,9 +565,6 @@ var Editor = React.createClass({
 
         localStorage.perseusLastCopiedText = selectedText;
         localStorage.perseusLastCopiedWidgets = JSON.stringify(widgetData);
-
-        console.log(
```

Comment:
@alopatin I don't really know how perseus copying works, would this be useful in prod?